cluster/mesos/task.go

```diff
@@ -184,7 +184,11 @@ func (t *task) monitor() (bool, []byte, error) {
 	case mesosproto.TaskState_TASK_FINISHED:
 		return true, taskStatus.Data, nil
 	case mesosproto.TaskState_TASK_FAILED:
-		return true, nil, errors.New(taskStatus.GetMessage())
+		errorMessage := taskStatus.GetMessage()
+		if errorMessage == "Abnormal executor termination" {
+			errorMessage += " : please verify your SWARM_MESOS_USER is correctly set"
```

Comment:
Please add a comment here at least then so that we know this is a blanket assumption we have and is actually masking other possible errors, also a todo to fix this.